web/styles/input_pill.css

```diff
@@ -376,6 +378,24 @@
     }
 }
 
+#user-profile-modal {
+    .user-type-custom-field-pill-container {
+        display: inline-flex;
+        gap: var(--vertical-spacing-input-pill)
+            var(--horizontal-spacing-input-pill);
+        max-width: 100%;
+        flex-wrap: wrap;
+
+        .pill-container {
+            max-width: 100%;
+        }
+
+        .pill-value {
+            max-width: 100%;
```

Comment:
Would be good to add a comment here and above for `pill-container` as well to make it clear this is used to override the `max-width` value set elsewhere.